pkg/registry/client.go

```diff
@@ -88,6 +88,39 @@ type (
 	ClientOption func(*Client)
 )
 
+// TODO remove in Helm 4
+// Deprecated
+// Fix for regression in v3.18.0
+// ORAS v1 was tolerant of an empty config file
+// ORAS v2 is not
+// to workaround this regression, ensure configfile is valid JSON
+func handleEmptyConfgFile(credentialsFile string, client *Client) {
```

Comment:
There is no test coverage for this.

Reply:
it's almost as if that was already acknowledged with a plan if we go forward with this approach https://github.com/helm/helm/pull/30924#issuecomment-2916584926